frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -54,6 +55,41 @@ const canDropEdgeOnNode = (operation: string, edge: Edge, node: Node): boolean =
 };
 
 const highlightNode = (monitor: DropTargetMonitor, props: NodeProps): boolean => {
+  if (
+    _.get(props, 'element.type') === 'knative-service' ||
+    (monitor.getItem() && monitor.getItem().type === 'event-source-link')
```

Comment:
use the `constants` here  for the types(TYPE_EVENT_SOURCE_LINK, TYPE_KNATIVE_SERVICE)